packages/orm/package.json

```diff
@@ -25,7 +25,8 @@
     "knex": "^3.1.0",
     "object-hash": "^3.0.0",
     "pg": "^8.11.5",
-    "pg-types": "^4.0.2"
+    "pg-types": "^4.0.2",
+    "temporal-polyfill": "^0.2.4"
```

Comment:
Wdyt of having this be a `devDependencies` in `joist-orm`, and have downstream projects that want to use it, pull it in explicitly? Looks like currently we're only using it to bumped timestamp fields...

Reply:
We use `toTemporalInstant` in `serde.ts`, so I don't think that's possible.

We also use `Temporal.PlainDate.from` in `newTestInstance`